examples/python-cicd-with-cloudbuilder/README.md

```diff
@@ -1,37 +1,56 @@
-# Basic Python CI With Cloud Source Repositories and Cloud Build
+# Basic Python Continuous Integration (CI) With Cloud Source Repositories (CSR) and Cloud Build
 
 ## Overview
-This repo contains code and instructions for using Google Cloud Source Repositories and Cloud Build to execute unit tests upon code check-in.
-
-The basic steps we need to follow are:
-1. Create a new project in Cloud Source Repository and clone it to your machine
-2. Create your code and unit tests
-3. Create a Python Cloud Builder (more below)
-4. Create a cloud build trigger (more below)
-5. Create a cloudbuild.yaml file that executes your tests using your python cloud builder.
-
-## 1. Create a new project in Cloud Source Repository
-* Go to https://source.cloud.google.com/
-* Click 'Add repository'
-* Choose 'Create new repository'
-* Specify a name, and your project name
-* Follow the instructions to 'git clone' the empty repo to your workstation and use it like you would a git repo
+This repo contains example code and instructions that show you how to use CSR and 
+Cloud Build to automatically run unit tests and pylint upon code check-in. By following this tutorial you will learn
+how to build basic Python continuous integration pipelines on Google Cloud Platform (GCP).
+
+By following along with this example you will learn how to:
+1. Create a new project in CSR and clone it to your machine.
```

Comment:
In markdown you should be able to just make all these "1." and when it renders it will count them appropriately, this makes editing way easier since you don't have to renumber everything as you work.

Reply:
I didn't know that!  Very cool

Reply:
Wouldn't this make reading the native document harder though?  I'll do whatever, just curious.

Reply:
People shouldn't be consuming markdown unrendered unless they're edititng it or something. Primary concern is a passive consumer, not an active contributor.